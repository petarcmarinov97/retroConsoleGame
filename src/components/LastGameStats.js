import React, { Component } from 'react';

export default class LastGameStats extends Component {
  render() {
    const { defaultStyle } = this.props;

    return <span id="lastGameStats" style={defaultStyle}></span>
  }
}
