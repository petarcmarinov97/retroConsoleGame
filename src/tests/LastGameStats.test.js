import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from "enzyme-to-json";
import LastGameStats from '../components/LastGameStats';

describe('LastGameStats tests', () => {

    let wrapper;
    const defaultStyle = {
        display: "none"
    }
    
    beforeEach(() => {
        wrapper = shallow(
            <LastGameStats defaultStyle={defaultStyle} />);
    })

    describe('Snapshots', () => {
        it("should render correctly", () => {
            expect(shallowToJson(wrapper)).toMatchSnapshot();
        })
    })
})