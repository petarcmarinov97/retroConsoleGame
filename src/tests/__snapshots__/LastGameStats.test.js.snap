// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LastGameStats tests Snapshots should render correctly 1`] = `
<span
  id="lastGameStats"
  style={
    Object {
      "display": "none",
    }
  }
/>
`;
